tests/gateway_helpers.go

```diff
@@ -0,0 +1,25 @@
+package tests
+
+// WritePath is the path to write influx 2.x points.
+const WritePath = "/api/v2/write"
```

Comment:
GoLand tells me this is unused

Reply:
Same, I removed all the unused constants.